src/main/java/Events.java

```diff
@@ -0,0 +1,12 @@
+public class Events extends Task {
```

Comment:
Perhaps the class should be called 'Event' instead of 'Events' as it's a class referencing a single object?